Library/Formula/srclib.rb

```diff
@@ -0,0 +1,150 @@
+require "language/go"
+
+class Srclib < Formula
+  homepage "https://srclib.org"
+  url "https://github.com/sourcegraph/srclib.git", :tag => "v0.0.37",
+      :revision => "de902433689c257ed87f6c7ee7abe14f6c146a73"
+  head "https://github.com/sourcegraph/srclib.git"
+
+  depends_on :hg => :build
+  depends_on "go" => :build
+
+  go_resource "code.google.com/p/rog-go" do
+    url "https://code.google.com/p/rog-go", :revision => "7088342b70fc1995ada4986ef2d093f340439c78", :using => :hg
```

Comment:
Might be nice to bump this onto multiple lines for the sake of readability, something like:

```ruby
go_resource "code.google.com/p/rog-go" do
  url "https://code.google.com/p/rog-go",
       :revision => "7088342b70fc1995ada4986ef2d093f340439c78", :using => :hg
end
```

Or something similar may work.